modules/core/04-channel/keeper/upgrade.go

```diff
@@ -702,7 +704,7 @@ func (k Keeper) WriteUpgradeTimeoutChannel(
 
 // startFlushing will set the upgrade last packet send and continue blocking the upgrade from continuing until all
 // in-flight packets have been flushed.
-func (k Keeper) startFlushing(ctx sdk.Context, portID, channelID string) error {
+func (k Keeper) startFlushing(ctx sdk.Context, portID, channelID string, upgrade *types.Upgrade) error {
```

Comment:
added a reference to the upgrade as the handler previously had a stale reference to the upgrade after changes were made within this function.